src/containers/password-and-security/PasswordAndSecurity.jsx

```diff
@@ -0,0 +1,149 @@
+import { Box, Typography } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+
+import AppButton from '~/components/app-button/AppButton'
+import AppCard from '~/components/app-card/AppCard'
+import AppTextField from '~/components/app-text-field/AppTextField'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { password as psw, confirmPassword } from '~/utils/validations/login'
+import { styles } from '~/containers/password-and-security/PasswordAndSecurity.styles'
+import useForm from '~/hooks/use-form'
+import useAxios from '~/hooks/use-axios'
+import { userService } from '~/services/user-service'
+import { useSelector } from 'react-redux'
+import useInputVisibility from '~/hooks/use-input-visibility'
+import { emptyField } from '~/utils/validations/common'
+import { useState } from 'react'
+
+const PasswordAndSecurity = () => {
+  const { t } = useTranslation()
+  const store = useSelector((state) => state.appMain)
+  const [newPasswordError, setNewPasswordError] = useState('')
+
+  const { fetchData: sendResetPassword } = useAxios({
+    service: (newPassword) => {
+      return userService.updateUser(store.userId, newPassword)
+    },
+    onResponseError: (error) => {
```

Comment:
Can we get other types of errors here besides the "same passwords" error? Maybe a validation error, for example if the BE has different rules.

Reply:
BE returns only "same password" error, there are no additional rules in updateUser to check password, so validations works only in frontend